bleak/utils.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+import re
+
+_address_separators = [":", "-"]
+_address_regex = re.compile(
+    rf"^([0-9A-Fa-f]{{2}}[{''.join(_address_separators)}]){{5}}([0-9A-Fa-f]{{2}})$"
+)
+
+
+def validate_address(address: str) -> bool:
```

Comment:
This function was not used anywhere in code, so I think we can just remove it completely.

Reply:
I  moved it back to where it was. It would be great to might add a check to BaseBleakClient to check the address for validity to get early feedback if you pass something wrong to the client. But this is not the topic of this pull request.